roles/ceph-client/meta/main.yml

```diff
@@ -1,11 +1,6 @@
 ---
 dependencies:
-  - role: monitoring-common
-    when: monitoring.enabled|default(True)|bool
-  - role: logging-config
-    when: logging.enabled|default(True)|bool
-    service: ceph
-    logdata: "{{ ceph.logs }}"
+  - role: ceph-config
```

Comment:
for ubuntu, we would not want to configure ceph repos on controller and compute nodes. The pkgs from ~bbc ceph repo collides with one that is provided by cloudarchive repo.